api/orchestration_movers.go

```diff
@@ -304,6 +328,10 @@ func (o *datasyncOrchestrator) createDatasyncLocation(ctx context.Context, mover
 			return nil
 		}); err != nil {
 			log.Infof("failed to create location, timeout retrying: %s", err.Error())
+
+			// clean up the role we created earlier
+			o.deleteBucketAccessRole(ctx, aws.String(roleARN))
```

Comment:
check the error?

Reply:
I didn't check it since it doesn't matter (this is best effort cleanup)

Reply:
you dont want to log it?

Reply:
yeah, good point 👍